### DISCUSSION TITLE
[@types/nightwatch] missing few definition

### DISCUSSION THREAD

**Discussion by ravi005e** [DefinitelyTyped/DefinitelyTyped#59651] [Issues with a @types package]
1. getCurrentUrl([callback])
2. .getNextSibling(selector, callback)
3. getPreviousSibling()
4. getShadowRoot(selector, callback)

[a comment by an automated account is not shown]

**Reply by ravi005e** [reply]
any update?

**Reply by vaibhavsingh97** [reply]
It's available  in v2.3.0
